pkg/storage/v2/postgres/client.go

```diff
@@ -18,15 +18,14 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"log"
 	"time"
 
-	"cloud.google.com/go/alloydbconn/driver/pgxv4"
+	_ "github.com/lib/pq"
```

Comment:
Please delete this line.

Reply:
To use postgres, we need this line:

https://pkg.go.dev/github.com/lib/pq